### PR TITLE
Feat/fuse dollar

### PR DESCRIPTION
**Pull request by leonprou**
## Proposed changes

- subj

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by LiorAgnin — APPROVED** [review]
Looks awesome, liked the `ActionOnRelay` approach

**Comment by leonprou** [comment]
Thanks @LiorAgnin! can you approve again?

**Review by LiorAgnin — DISMISSED** [review]

File: server/config/default.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ module.exports = {
     },
     home: {
       name: 'fuse',
-      bridgeType: 'home',
+      bridgeType: 'mainnet',
```
Comment (LiorAgnin) [inline server/config/default.js:63]: Is this ok?

Comment (leonprou) [reply server/config/default.js:63]: @LiorAgnin my bad.. fixing

**Review by LiorAgnin — APPROVED** [review]

**Pull request merged**